react-native/training/src/app/screens/TodoList/components/List/components/Item/index.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { View, Text, TouchableOpacity } from 'react-native';
+import PropTypes from 'prop-types';
+
+import Checkbox from '../../../Checkbox';
+
+import styles from './styles';
+
+function Item({ index, item, onToggleItemCompleted, onRemoveItem }) {
+  const itemStyle = item.completed ? [styles.item, styles.completed] : styles.item;
```

Comment:
You could change this for: `const itemStyle =  ${styles.item} ${item.completed ? styles.completed : ''}`